### ISSUE TITLE
Change capitalization in header. 

### ISSUE THREAD

**Issue by Meggs23** [opensandiego/opensandiego.github.io#13]
@nikolajbaer ...

Please change 'we are OPEN SAN DIEGO' to...

We are Open San Diego

**Comment by nikolajbaer** [comment]
I assigned this to Evan, as he updated the capitalization initially. Please coordinate with him on the final version!

**Comment by evanshulman** [comment]
Hey hey, 

For context: yes, that first edits I submitted was just to get us started, and I was going for all caps since our logo uses all caps. I figured we would modify later/as we go. I don't have a strong opinion about it, but was mainly going for keeping a visual consistency, especially for the Header/Jumbotron which is really just about making a big visual splash, especially since we moved most of the readable content lower or to other pages.
